pkg/agent/profiles.go

```diff
@@ -228,7 +228,7 @@ func (tg *TargetGroup) targetsFromGroup(group *targetgroup.Group) ([]*Target, []
 				}
 
 				if pcfg, found := tg.config.ProfilingConfig.PprofConfig[profType]; found && pcfg.Delta {
-					params.Add("seconds", strconv.Itoa(int(time.Duration(tg.config.ScrapeTimeout)/time.Second)-1))
+					params.Add("seconds", strconv.Itoa(int(time.Duration(tg.config.ScrapeInterval)/time.Second)))
```

Comment:
I am not too sure why this had been removed. I always thought that this will make sure we have a consistent scrape time, even when network latency is added (up to 1 second).

Let's take this example:

scrape_interval: 15s
scrape_timeout: 18s
profiling_time: 15s (was 14s before) 

After this change, when we have network latency of 500ms, we would miss out on scrapes, because the real scrape interval would be 15s+network latency and this will create problems when we look at things over time e.g. this historgram screenshot (similar to the prometheus rate([twice_scrape_interval]) problem.

Here you can see a point missing at 11:12:47 because of that:


![scrn-2023-01-11-11-13-41](https://user-images.githubusercontent.com/223048/211792306-8bb078a0-89fd-45fd-ab80-c43a69120296.png)